SpaceDock/app.py

```diff
@@ -1,6 +1,10 @@
 from flask import Flask, render_template, request, g, Response, redirect, session, abort, send_file, url_for
-from flask_login import LoginManager, current_user
+from flask.ext.login import LoginManager, current_user
```

Comment:
From what I can tell of my 5 minutes of Flask experience, this is the outdated way for extension handling. This line should stay as is, and the following need changing to `flask_htmlmin` and `flask_cache` (although flask_cache is currently making trouble for me).

Reply:
Yup, that was a wrongly resolved conflict. I checked a few of these but should have been more systematic...